src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,34 +1,52 @@
 package edu.kis.vh.nursery;
 
 public class DefaultCountingOutRhymer {
+    private static final int DEFAULT_RETURN_VALUE = -1;
+    private static final int MAX_STACK_SIZE = 12;
+    private int[] numbers = new int[getMaxStackSize()];
 
-    private int[] numbers = new int[12];
+    private int total = getDefaultReturnValue();

```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)